ras_party/controllers/survey_controller.py

```diff
@@ -0,0 +1,31 @@
+import logging
+
+import requests
+import structlog
+from flask import current_app
+from requests.exceptions import ConnectionError, HTTPError, Timeout
+
+from ras_party.exceptions import ServiceUnavailableException
+
+logger = structlog.wrap_logger(logging.getLogger(__name__))
+
+
+def get_surveys_details() -> dict:
```

Comment:
In the future, I would like to make this a single app call, cache etc. At the moment this is fine